src/Styleguide/Pages/Artist/ArtistApp.tsx

```diff
@@ -1,13 +1,12 @@
-import { ArtistHeader_artist } from "__generated__/ArtistHeader_artist.graphql"
 import { RecentlyViewed_me } from "__generated__/RecentlyViewed_me.graphql"
 import React from "react"
 import { Footer } from "Styleguide/Components/Footer"
 import { RecentlyViewedFragmentContainer as RecentlyViewed } from "Styleguide/Components/RecentlyViewed"
-import { RouteTab, RouteTabs } from "Styleguide/Components/RouteTabs"
 import { Box } from "Styleguide/Elements/Box"
 import { Col, Row } from "Styleguide/Elements/Grid"
 import { Separator } from "Styleguide/Elements/Separator"
 import { Spacer } from "Styleguide/Elements/Spacer"
+import { TabsFragmentContainer } from "Styleguide/Pages/Artist/Tabs"
```

Comment:
```
import { TabsFragmentContainer as Tabs } from "Styleguide/Pages/Artist/Tabs"
```

Reply:
👍

Reply:
Actually, can this be renamed `NavigationTabs`? Tabs seems a bit ambiguous